examples/CertificateComplexExample/es6.js

```diff
@@ -24,127 +26,112 @@ const crls = []; // Array of CRLs for all certificates (trusted + intermediate)
+	for (const rdn of certificate.issuer) {
+		for (const typeAndValue of rdn) {
+			let typeval = AttributeTypeDictionnary[typeAndValue.type];
+			if (typeof typeval === "undefined")
+				typeval = typeAndValue.type;
+
+			const subjval = typeAndValue.value.valueBlock.value;
+
+			const row = issuerTable.insertRow(issuerTable.rows.length);
+			const cell0 = row.insertCell(0);
+			// noinspection InnerHTMLJS
+			cell0.innerHTML = typeval;
+			const cell1 = row.insertCell(1);
+			// noinspection InnerHTMLJS
+			cell1.innerHTML = subjval;
+		}
 	}
 	//endregion
-	
+
 	//region Put information about X.509 certificate subject
-	for(const typeAndValue of certificate.subject.typesAndValues)
-	{
-		let typeval = rdnmap[typeAndValue.type];
-		if(typeof typeval === "undefined")
-			typeval = typeAndValue.type;
-		
-		const subjval = typeAndValue.value.valueBlock.value;
-		
-		const row = subjectTable.insertRow(subjectTable.rows.length);
-		const cell0 = row.insertCell(0);
-		// noinspection InnerHTMLJS
-		cell0.innerHTML = typeval;
-		const cell1 = row.insertCell(1);
-		// noinspection InnerHTMLJS
-		cell1.innerHTML = subjval;
+	for (const rdn of certificate.subject) {
+		for (const typeAndValue of rdn) {
+			let typeval = AttributeTypeDictionnary[typeAndValue.type];
+			if (typeof typeval === "undefined")
+				typeval = typeAndValue.type;
+
+			const subjval = typeAndValue.value.valueBlock.value;
+
+			const row = subjectTable.insertRow(subjectTable.rows.length);
+			const cell0 = row.insertCell(0);
+			// noinspection InnerHTMLJS
+			cell0.innerHTML = typeval;
+			const cell1 = row.insertCell(1);
+			// noinspection InnerHTMLJS
+			cell1.innerHTML = subjval;
+		}
 	}
 	//endregion
-	
+
 	//region Put information about X.509 certificate serial number
 	// noinspection InnerHTMLJS
 	document.getElementById("cert-serial-number").innerHTML = bufferToHexCodes(certificate.serialNumber.valueBlock.valueHex);
 	//endregion
-	
+
 	//region Put information about issuance date
 	// noinspection InnerHTMLJS
 	document.getElementById("cert-not-before").innerHTML = certificate.notBefore.value.toString();
 	//endregion
-	
+
 	//region Put information about expiration date
 	// noinspection InnerHTMLJS
 	document.getElementById("cert-not-after").innerHTML = certificate.notAfter.value.toString();
 	//endregion
-	
+
 	//region Put information about subject public key size
 	let publicKeySize = "< unknown >";
-	
-	if(certificate.subjectPublicKeyInfo.algorithm.algorithmId.indexOf("1.2.840.113549") !== (-1))
-	{
+
+	if (certificate.subjectPublicKeyInfo.algorithm.algorithmId.indexOf("1.2.840.113549") !== (-1)) {
 		const asn1PublicKey = asn1js.fromBER(certificate.subjectPublicKeyInfo.subjectPublicKey.valueBlock.valueHex);
 		const rsaPublicKey = new RSAPublicKey({ schema: asn1PublicKey.result });
-		
+
 		const modulusView = new Uint8Array(rsaPublicKey.modulus.valueBlock.valueHex);
 		let modulusBitLength = 0;
-		
-		if(modulusView[0] === 0x00)
+
+		if (modulusView[0] === 0x00)
```

Comment:
Please format the document using linter rulles

Reply:
I added some rules to match your global rules in `.eslintrc.js`. 
In addition, I added an `.editorconfig` for those having a compatible IDE.